src/app/tasks.service.ts

```diff
@@ -0,0 +1,100 @@
+import { Injectable } from '@angular/core';
+import { Observable, from } from 'rxjs';
+
+export interface ActionTasksElement {
+  title: string;
+  projectName: string;
+  type: string;
+  completed: string;
+  dueDate: object;
+  dueDay: number;
+  color: string;
+}
+
+@Injectable({
+  providedIn: 'root'
+})
+export class TasksService {
+  constructor() {}
+  tableData: ActionTasksElement[] = [
```

Comment:
This should be private readonly. https://www.typescriptlang.org/docs/handbook/classes.html#understanding-private